yb-voyager/cmd/export.go

```diff
@@ -229,6 +238,23 @@ func validateOracleParams() {
 	} else if source.DBSid != "" {
 		utils.PrintAndLog("Using SID for export.")
 	}
+	if source.IsOracleCDBSetup() {
```

Comment:
How to convey that we don't need these CLI arguments for offline migration.

Reply:
Can add it to the command help doc. Additionally we can also add it to our voyager documentation on the web.